receiver/couchdbreceiver/factory_test.go

```diff
@@ -45,13 +45,13 @@ func TestValidConfig(t *testing.T) {
 
 func TestCreateMetricsReceiver(t *testing.T) {
 	factory := NewFactory()
-	_, err := factory.CreateMetricsReceiver(
+	metricsReceiver, err := factory.CreateMetricsReceiver(
```

Comment:
We should have a test, probably this one, that creates the receiver using the config from `createDefaultConfig`. If necessary, assign values to it, but at least get the default at a starting point.